components/tenant-mgt/org.wso2.carbon.tenant.keystore.mgt/src/main/java/org/wso2/carbon/keystore/mgt/KeyStoreGenerator.java

```diff
@@ -302,4 +307,17 @@ private String getTenantDomainName() throws KeyStoreMgtException {
             throw new KeyStoreMgtException(msg, e);
         }
     }
+
+    /**
+     * This method returns the preferred JCE provider identifier to be used.
+     *
+     * @return jce provider identifier name
+     */
+    private String getPreferredJceProviderIdentifier() {
+        String provider = System.getProperty("security.jce.provider");
```

Comment:
Move this property to a constant

Reply:
Done with https://github.com/wso2/carbon-multitenancy/pull/237/commits/0fbe579ca2adf9379f0d0eb64f721f86d6ebc0a0